drag_and_drop_v2/templates/html/drag_and_drop_edit.html

```diff
@@ -67,15 +89,21 @@
             </header>
             <section class="tab-content">
                 <form class="target-image-form">
-                    <label class="h3" for="background-url">{% trans "Background URL" %}</label>
-                    <input id="background-url"
+                    <label class="h3" for="background-url-{{id_suffix}}">
+                        <span>{% trans "Background URL" %}</span>
+                    </label>
+                    <input class="background-url"
+                           id="background-url-{{id_suffix}}"
                            type="text"
                            placeholder="{% trans 'For example, http://example.com/background.png or /static/background.png' %}">
```

Comment:
Placeholders should be avoided-- they are too low in contrast, and they don't provide a very good user experience because the text disappears when you start typing. It's better to just add a visible (to all) description.